File: app/src/main/java/com/hazem/wazaker/Data_sets/AzkarListDataset.java
package com.hazem.wazaker.Data_sets;

import com.hazem.wazaker.Models.AzkarModel;

import java.util.ArrayList;
import java.util.List;

public class AzkarListDataset {

    public static List<AzkarModel> azkarList;

    public static List<AzkarModel> getAzkarList() {

        azkarList = new ArrayList<>();
        azkarList.add(new AzkarModel("أذكار المسجد"));
        azkarList.add(new AzkarModel("أذكار بعد الصلاة "));
        azkarList.add(new AzkarModel("أذكار الأذان "));
        azkarList.add(new AzkarModel("أذكار الوضوء"));
        azkarList.add(new AzkarModel("الأدعيـة القرآنية"));
        azkarList.add(new AzkarModel("الأدعية النبوية "));
        azkarList.add(new AzkarModel("أذكار الطعام"));
        azkarList.add(new AzkarModel("أذكار المنزل"));
        azkarList.add(new AzkarModel("أذكار النوم"));
        azkarList.add(new AzkarModel("أذكار الإستيقاظ من النوم"));
        azkarList.add(new AzkarModel("أذكار عظيمة"));
        azkarList.add(new AzkarModel("دعاء السفر"));
        azkarList.add(new AzkarModel("دعاء ختم القرآن الكريم"));
        azkarList.add(new AzkarModel("فضل الدعاء"));
        azkarList.add(new AzkarModel("فضل تلاوة القرآن الكريم"));
        azkarList.add(new AzkarModel("فضل الأذكار"));
        azkarList.add(new AzkarModel("الرقية الشرعية"));
        azkarList.add(new AzkarModel("أدعية للمتوفّي"));

        return azkarList;
    }}

File: app/src/main/java/com/islamic/Wazaker/Data/DataSets/AzkarListDataset.kt
package com.islamic.Wazaker.Data.DataSets

import com.islamic.Wazaker.Data.Models.AzkarListModel

object AzkarListDataset {
    private lateinit var azkarList: MutableList<AzkarListModel>

    fun getAzkarList(): List<AzkarListModel> {
        azkarList = ArrayList()
        azkarList.add(AzkarListModel("أذكار المسجد"))
        azkarList.add(AzkarListModel("أذكار بعد الصلاة "))
        azkarList.add(AzkarListModel("أذكار الأذان "))
        azkarList.add(AzkarListModel("أذكار الوضوء"))
        azkarList.add(AzkarListModel("الأدعيـة القرآنية"))
        azkarList.add(AzkarListModel("الأدعية النبوية "))
        azkarList.add(AzkarListModel("أذكار الطعام"))
        azkarList.add(AzkarListModel("أذكار المنزل"))
        azkarList.add(AzkarListModel("أذكار النوم"))
        azkarList.add(AzkarListModel("أذكار الإستيقاظ من النوم"))
        azkarList.add(AzkarListModel("أذكار عظيمة"))
        azkarList.add(AzkarListModel("دعاء السفر"))
        azkarList.add(AzkarListModel("دعاء ختم القرآن الكريم"))
        azkarList.add(AzkarListModel("فضل الدعاء"))
        azkarList.add(AzkarListModel("فضل تلاوة القرآن الكريم"))
        azkarList.add(AzkarListModel("فضل الأذكار"))
        azkarList.add(AzkarListModel("الرقية الشرعية"))
        azkarList.add(AzkarListModel("أدعية للمتوفّي"))

        return azkarList
    }
}